yattag/indentation.py

```diff
@@ -11,34 +17,39 @@
 
 class TokenMeta(type):
 
-    _token_classes = {}
+    _token_classes = {}  # type: Dict[str, Any]
 
     def __new__(cls, name, bases, attrs):
+        # type: (str, Tuple[Any], Dict[str, Any]) -> Any
         kls = type.__new__(cls, name, bases, attrs)
         cls._token_classes[name] = kls
         return kls
 
     @classmethod
     def getclass(cls, name):
+        # type: (str) -> Any
         return cls._token_classes[name]
 
 # need to proceed that way for Python 2/3 compatility:
-TokenBase = TokenMeta('TokenBase', (object,), {})
+TokenBase = TokenMeta('TokenBase', (object,), {}) # type: Any
```

Comment:
````
error: Invalid type "yattag.indentation.TokenBase"
````

is what you get without it. There is doc on how to use metaclasses with mypy here: https://mypy.readthedocs.io/en/latest/metaclasses.html, but it's not obvious to me how to leave your tricky code unchanged (so that it keeps working with python2/3). So I just left it as Any for now.